pkg/sdn/plugin/pod_linux.go

```diff
@@ -493,12 +498,18 @@ func (m *podManager) update(req *cniserver.PodRequest) (*runningPod, error) {
 		return nil, err
 	}
 
+	ofport, err := m.ovs.GetOFPort(hostVethName)
+	if err != nil {
+		return nil, err
+	}
+
 	return &runningPod{
 		activePod: &kubehostport.ActivePod{
 			Pod: pod,
 			IP:  net.ParseIP(podIP),
 		},
-		vnid: podConfig.vnid,
+		vnid:   podConfig.vnid,
+		ofport: ofport,
```

Comment:
Is there some other way update() could be handled, so you didn't have to re-call GetOFPort() (and re-parse podIP) given that they're both constant over the life of the pod? You're creating a new runningPod object every time just to deal with the fact that vnid can change, but maybe you should just return a new vnid instead.